app/javascript/components/pages/FateNew.js

```diff
@@ -1,11 +1,76 @@
 import React,{Component} from 'react'
+import {Form, FormGroup, Input, Label, Button} from 'reactstrap'
+
 
 class FateNew extends Component{
+  constructor(props){
+    super(props)
+
+    this.state = {
+      form: {
+        category: "",
+        url: "",
+        description: "",
+        is_public: "",
+      },
+      submitted: false
+    }
+  }
+
+  handleChange = (e) => {
+        // destructuring form out of state
+        let { form } = this.state
```

Comment:
prefer `const` if you don't need mutation. just good practice.

Reply:
Will update it to change, i think with the variables inside of functions they've been having us use let, will update here and try to be more careful mvoing on